### DISCUSSION TITLE
Wanting to contribute, not sure what's already been worked on

### DISCUSSION THREAD

**Discussion by X-lem** [docmost/docmost#306] [Help / Questions]
Hello 👋 

I'm interested in contributing, but don't want to waste time on something someone else is already working on. Is there a place that lists things that are already being worked on? Or a guide on features that you plan on adding, but have not been started yet? I've looked on the website/github/docs pages but don't see anything.

I see a number of feature requests in discussions I could potentially pickup, but I would imagine some of them are already being worked on already.

**Comment by Philipinho** [comment]
I will make this better in the near future.
Would you like to work on https://github.com/docmost/docmost/issues/93?

**Reply by X-lem** [reply]
@Philipinho I think I can do this one. I was tinkering with the code base this weekend trying to figure everything out. Might have some questions about it at some point. Where's the best spot to send those if they come up? Here?

**Reply by Philipinho** [reply]
@X-lem, feel free to send any queries to the email on my profile.
I appreciate your interest in contributing.

**Comment by mustafa-yilmaz** [comment]
Being a very small group of Docmost users, perhaps we (our group/team) may not be suitable to be assigned to (matched with) all free (available) development tasks but we would be so glad to contribute to the Docmost initiative by solving our issues that we currently and will face in order to speed the processes up as we also expressed at  #651  . Thanks.
